notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb

```diff
@@ -1276,7 +1276,7 @@
       },
       "outputs": [],
       "source": [
-        "%%bash --out ENDPOINT_MODEL_ID\n",
+        "%%bash -s \"$ENDPOINT_ID\" --out ENDPOINT_MODEL_ID\n",
         "ENDPOINT_MODEL_ID=$(gcloud ai endpoints describe 4308018499021701120 --region=us-central1 --project=jfacevedo-project | grep \"id:\")\n",
```

Comment:
Do you also want to generalize the hardcoded endpoint ID and project in this command? (And region perhaps?)

Reply:
done. Thanks!